source/game.py

```diff
@@ -1,22 +1,101 @@
+import string
+from os import path
+
+from blessed import Terminal
```

Comment:
```suggestion
from blessed import Terminal as Interface
```
We use Interface instead of Terminal